scripts/release.sh

```diff
@@ -72,6 +72,16 @@ export RELEASE_VER=$(sed -n '/^__version__/p' $INIT_FILE | cut -d \' -f2)
 LAST_VER=$(git tag -l | sort -V | tail -n1)
 printf "last version: \e[1;32m$LAST_VER\e[0m\n"
 
+# Update new _versions.yml and _versions.json if necessary
+if [[ ! $(head -n1 ./docs/_versions.yml) == "- version: v$RELEASE_VER" ]]; then
+    printf -- "- version: v$RELEASE_VER\n" > ./docs/new_versions.yml
+    cat ./docs/_versions.yml >> ./docs/new_versions.yml
+    mv ./docs/new_versions.yml ./docs/_versions.yml
+    python ./scripts/versions_yaml_2_json.py
+    git add ./docs/_versions.yml
```

Comment:
Why do we keep json and yaml?

Reply:
The yaml is more human readable and the json is more machine readable.
`_versions.yml` can go and` _versions.json` stay and everything will still work.
Just it's a pain to edit the `_versions.json` by hand compared to the `_versions.yml`

Reply:
why does it need to be edited by hand?

Reply:
We might want to change the versions we support documentation for

Reply:
But should be pretty rare. Maybe it's not worth the added complexity. Just have the person run the Python REPL to edit the JSON if we ever need it.

Reply:
yes, let's have a single source of truth